agent/custom/bbc_connection_manager.py

```
@@ -28,42 +28,95 @@
+        
+        # 终止BBC进程
+        if self._state.get('bbc_process'):
+            try:
+                self._kill_bbc_process()
+            except:
+                pass
+        
+        time.sleep(0.5)
+        mfaalog.info("[BbcConnectionManager] 旧状态已清理")
+    
+    def _cleanup_port(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_cleanup_port`

Add return type annotation: `None`

(ANN202)